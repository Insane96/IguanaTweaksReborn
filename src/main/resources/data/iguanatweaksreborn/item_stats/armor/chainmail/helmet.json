{
	"item": "minecraft:chainmail_helmet",
	"durability": 52,
	"armor": 4,
	"toughness": 1
}