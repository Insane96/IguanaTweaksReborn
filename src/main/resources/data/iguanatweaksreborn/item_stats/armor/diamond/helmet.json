{
	"item": "minecraft:diamond_helmet",
	"durability": 195,
	"armor": 2,
	"toughness": 1,
	"knockback_resistance": 0.03
}