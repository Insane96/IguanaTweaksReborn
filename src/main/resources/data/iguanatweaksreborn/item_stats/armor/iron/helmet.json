{
	"item": "minecraft:iron_helmet",
	"durability": 78,
	"armor": 2
}