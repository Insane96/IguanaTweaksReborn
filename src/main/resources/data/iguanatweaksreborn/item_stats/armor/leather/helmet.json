{
	"item": "minecraft:leather_helmet",
	"durability": 78,
	"armor": 1,
	"armor_toughness": 1
}