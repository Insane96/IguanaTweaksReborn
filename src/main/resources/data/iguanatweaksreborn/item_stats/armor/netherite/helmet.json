{
	"item": "minecraft:netherite_helmet",
	"durability": 182,
	"armor": 4,
	"toughness": 2,
	"knockback_resistance": 0.05
}