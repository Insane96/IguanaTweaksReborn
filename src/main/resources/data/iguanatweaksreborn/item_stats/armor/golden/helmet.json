{
	"item": "minecraft:golden_helmet",
	"durability": 78,
	"armor": 1,
	"toughness": 1
}